<launch>
<node pkg="bluetooth_sensors" type="ble_interface.py" name="ble_interface_node" output="screen"></node>
</launch>
